packages/taquito-core/src/taquito-core.ts

```diff
@@ -3,4 +3,7 @@
  * @module @taquito/core
  */
 
+import stringify from 'json-stringify-safe';
+
 export * from './errors';
+export { stringify };
```

Comment:
see `stringify` import and export didn't see it used anywhere

Reply:
I created this issue: #2672 for this, and undid the change here.